.travis.yml

```diff
@@ -0,0 +1,25 @@
+# use Docker-based container (instead of OpenVZ)
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.m2/repository
+    - $HOME/.sbt
+    - $HOME/.ivy2
+
+language: scala
+
+script:
+  - sbt ++$TRAVIS_SCALA_VERSION test
+
+  # Trick to avoid unnecessary cache updates
+  - find $HOME/.sbt -name "*.lock" | xargs rm
```

Comment:
it's the good practice I got from scalaz :palm_tree: their recommendations are usually solid :wheelchair: 

See [https://github.com/scalaz/scalaz/blob/series/7.2.x/.travis.yml]